doc/content/examples/example3.md

```diff
@@ -87,4 +87,15 @@ DEEPSOIL and presented in
 Further details regarding to application of MASTODON to simulate seismic site response and soil
 structure interaction analysis can be found in [!citet](baltaji2017nonlinear).
 
+This example can also be run using the Automatic Differentiation (AD). Automatic Differentiation
+computes the "exact" empirical Jacobian of the stiffness matrix as opposed to the standard
```

Comment:
what is an 'empirical Jacobian of the stiffness matrix'? Do you mean numerical? And its not a Jacobian of a stiffness matrix -- its just Jacobian.  

Please change this to a 'numerical Jacobian that is very close to the theoretical Jacobian.'